Rubberduck.VBEEditor/ComManagement/TypeLibsAPI.cs

```diff
@@ -11,11 +11,11 @@ namespace Rubberduck.VBEditor.ComManagement.TypeLibsAPI
 {
     public class VBETypeLibsAPI
     {
-        public static void ExecuteCode(IVBE ide, string projectName, string standardModuleName, string procName, object[] args = null)
+        public static object ExecuteCode(IVBE ide, string projectName, string standardModuleName, string procName, object[] args = null)
```

Comment:
I think we will also need documentation on wha to do with the returned `object` -- are we responsible for releasing it and under what conditions (e..g if it's not a primitive type but an actual class object?) to ensure we don't leak the returned value.

Reply:
Does `ExecuteCode` get around the problem in Access, where `Sub Foo` is defined in `Module1` **and** `Module2`, where `Application.Run "Module2.Foo"` would result in:
> Run-time error '2517' : 
> Microsoft Access cannot find the procedure 'Module2.Foo.'

Reply:
Hi @ThunderFrame, yes it does